picard/ui/mainwindow.py

```diff
@@ -147,6 +147,9 @@ def keyPressEvent(self, event):
     def show(self):
         self.restoreWindowState()
         super().show()
+        if config.setting['check_for_updates'] and config.setting['update_check_days'] > 0 and datetime.date.today().toordinal() >= config.persist['last_update_check'] + config.setting['update_check_days']:
+            log.debug(_("Initiating start-up check for program updates."))
```

Comment:
May be add info concerning conditions in this debug message, like last check date, etc...

Reply:
Changed to log a line showing:
- whether the automatic check was initiated or skipped
- today's date
- date of last check
- checking interval
- update check subscription level